unified-installer/astra-unified-installer.sh

```diff
@@ -1829,7 +1834,7 @@ step_check_k8s_permissions() {
     if [ "$has_permission" == "yes" ]; then
         logdebug "k8s permissions: OK"
         return 0
-    elif [ "$has_permission" != "yes" ]; then
+    elif echo "$has_permission" | grep -q "no"; then
```

Comment:
This check was too loose; an empty value would manifest as the user not having permission, when it actually means there's an error.